### ISSUE TITLE
issue with Autotools

### ISSUE THREAD

**Issue by dcopetti** [bcgsc/arcs#58]
Hello,

I am trying to install ARCS and I have this issue with Automake:
```
[copettid@kp141-242 arcs]$ ./autogen.sh
+ aclocal
threads object version 2.21 does not match bootstrap parameter 2.16 at /usr/lib64/perl5/DynaLoader.pm line 210.
Compilation failed in require at /usr/share/automake-1.15/Automake/ChannelDefs.pm line 23.
BEGIN failed--compilation aborted at /usr/share/automake-1.15/Automake/ChannelDefs.pm line 26.
Compilation failed in require at /usr/share/automake-1.15/Automake/Configure_ac.pm line 27.
BEGIN failed--compilation aborted at /usr/share/automake-1.15/Automake/Configure_ac.pm line 27.
Compilation failed in require at /usr/bin/aclocal line 39.
BEGIN failed--compilation aborted at /usr/bin/aclocal line 39.
[copettid@kp141-242 arcs]$ cd ..
[copettid@kp141-242 bin]$ sudo dnf install autoconf automake
[sudo] password for copettid:
Last metadata expiration check: 2:45:43 ago on Fri 29 Mar 2019 09:19:18 AM CET.
Package autoconf-2.69-27.fc28.noarch is already installed, skipping.
Package automake-1.15.1-5.fc28.noarch is already installed, skipping.
Dependencies resolved.
Nothing to do.
Complete!

```
It should be there, right? I don't understand what is wrong at this point.

**Comment by sjackman** [comment]
Hi, @dcopetti. There's no need to run `./autogen.sh`. Download the released tarball `arcs-1.0.5.tar.gz
` from the Releases page https://github.com/bcgsc/arcs/releases. Here's a direct link: https://github.com/bcgsc/arcs/releases/download/v1.0.5/arcs-1.0.5.tar.gz
Extract the tarball and then run…
```sh
curl -LO https://github.com/bcgsc/arcs/releases/download/v1.0.5/arcs-1.0.5.tar.gz
tar xf arcs-1.0.5.tar.gz
cd arcs-1.0.5
./configure --prefix=$HOME/opt/arcs && make && make install
PREFIX=$HOME/opt/arcs/bin:$PATH
ARCS --version
```

Even better is to install ARCS using a package manager such as Homebrew (on macOS or Linux) or Bioconda. See http://www.birollab.ca/software.html

Install Homebrew on macOS: https://brew.sh
Install Homebrew on Linux: https://docs.brew.sh/Homebrew-on-Linux
Then run `brew install brewsci/bio/arcs`

**Comment by sjackman** [comment]
Bioconda does not currently have a formula for ARCS.
@bcgsc/btl It'd be pretty straight forward to add. If someone would like to gain experience adding a recipe to Bioconda, I can point you in the right direction.
https://github.com/bioconda/bioconda-recipes/tree/master/recipes/arcs

Here's the recipe for ABySS.
https://github.com/bioconda/bioconda-recipes/tree/master/recipes/abyss
